testing/bdd/features/timing/bufferDelayNode.feature

```diff
@@ -15,20 +15,24 @@ Feature: BufferDelayNode
   Scenario: BufferDelayNode delays emission by no less than the delay period
     Given an xml file <xml_file>
     And the document is generated
-    And the document has availability time <availability_1>  
-    And the document is delayed by <delay_offset>
+    And it has availability time <avail_time>
+    And the buffer delay node delays it by <delay_offset>
     And the document is emitted at <emission>
-    Then the delta between emission and availability_1 is greater or equal to <delay_offset>
+    Then the delta between <emission> and <avail_time> is greater or equal to <delay_offset>
+
+    Examples:
+    | emission   | avail_time  | delay_offset |
+    | 00:00:05.0 | 00:00:02.0  | 00:00:01.0   |
 
 
   # SPEC-CONFORMANCE.md R110, R111
   Scenario: BufferDelayNode does not modify the document
```

Comment:
I think we should remove this scenario altogether, since the spec does permit appliedProcessing to be "silently" added, so it does not even require that the input document is identical to the output document.